Server/models/page.js

```diff
@@ -0,0 +1,29 @@
+const { DataTypes } = require('sequelize');
+const sequelize = require('../util/database');
+
+const Page = sequelize.define('Page', {
+    PageNumber: {
+        type: DataTypes.INTEGER,
+        autoIncrement: true,
+        allowNull: false,
+        primaryKey: true
+    },
+    PageId: {
+        type: DataTypes.INTEGER,
+    },
+    BookId: {
+        type: DataTypes.INTEGER
+    },
+    ImageURL: {
+        type: DataTypes.STRING
+    },
+    Text: {
+        type: DataTypes.STRING,
+    },
+    Caption: {
+        type: DataTypes.STRING,
+    }
+})
+
```

Comment:
Add those two lines of code to set up the relationship with Book. See Book and User models for an example.